shared/bash_remediation_functions/fix_audit_syscall_rule.sh

```diff
@@ -111,7 +113,8 @@ do
 	# * follow the rule pattern, and
 	# * meet the hardware architecture requirement, and
 	# * are current syscall group specific
-	IFS=$'\n' existing_rules=($(sed -e "/${pattern}/!d" -e "/${arch}/!d" -e "/${group}/!d"  "$audit_file"))
+	IFS=$'\n' existing_rules=($(sed -e ";${pattern};!d" -e "/${arch}/!d" -e "/${group}/!d"  "$audit_file"))
+	retval=$?
```

Comment:
why is it here but not on line 87, what is this about (nothing in commit message, no comments)